src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -546,27 +550,18 @@ private static bool IsNotPlayOffThemeAndBothTeamsNotNull(Game newGame, Tournamen
             TournamentScheduleDto tournamentScheduleInfo)
         {
             string errorMessage = null;
-            if (GameValidation.IsFreeDayGame(newGame))
-            {
-                if (tournamentScheduleInfo.Scheme != TournamentSchemeEnum.PlayOff)
-                {
-                    errorMessage = Resources.SameFreeDayGameInRound;
-                }
-                else
-                {
-                    errorMessage = string.Format(
+            errorMessage = GameValidation.IsFreeDayGame(newGame)
+                ? (tournamentScheduleInfo.Scheme !=
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3358)